rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -85,6 +117,7 @@ class BridgeSupportFlyoverTest {
     private FederationSupportBuilder federationSupportBuilder;
     private ActivationConfig.ForBlock activations;
     private SignatureCache signatureCache;
+    private LockingCapSupport lockingCapSupport;
```

Comment:
nit: i'd move the variable declaration to above, right after the `whitelistSupport` one

Reply:
Done